### PR TITLE
Clean up apt lists after installation in Dockerfile

### PR DESCRIPTION
**Pull request by PeterDaveHello**
Kind of Dockerfile best practices 🤩 

Image size change as below:
```
REPOSITORY          TAG                  IMAGE ID       CREATED          SIZE
after               latest               de7531413c1e   6 minutes ago    463MB
before              latest               f2756a4fd85a   12 minutes ago   481MB
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,8 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install -j$(nproc) iconv \
     && docker-php-ext-configure gd --with-freetype=/usr/include/ --with-jpeg=/usr/include/ \
     && docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql \
-    && docker-php-ext-install -j$(nproc) gd pdo pdo_mysql pdo_pgsql pgsql
+    && docker-php-ext-install -j$(nproc) gd pdo pdo_mysql pdo_pgsql pgsql \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (kniziol) [inline Dockerfile:13]:
```suggestion
    && rm -rf \
        /var/lib/apt/lists/* \
        /tmp/* \
        /var/tmp/*
```

**Review by kniziol — COMMENTED** [review]
We could also remove more temporary data

File: Dockerfile (hunk 1 above)
Comment (PeterDaveHello) [reply Dockerfile:13]: `apt-get clean` is actually not needed :smile:

Comment (kniziol) [reply Dockerfile:13]: Why?

According to the https://manpages.ubuntu.com/manpages/lunar/en/man8/apt-get.8.html documentation page, the `apt-get clean` command removes package files from:
- `/var/cache/apt/archives/`
- `/var/cache/apt/archives/partial/`

directories that have been cleared neither by `rm -rf` nor another command 🙃

Comment (sstidl) [reply Dockerfile:13]: i checked it in the image. /tmp and /var/tmp are empty. also the cache dir is slim.
it's not worth deleting more than `/var/lib/apt/lists/*`

**Review by sstidl — APPROVED** [review]

**Pull request merged**